<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TempoTransformOptionWidget</class>
 <widget class="QWidget" name="TempoTransformOptionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>215</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,10,0">
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>From</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="fromDoubleSpinBox">
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>99999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="guessSrcTempoButton">
       <property name="text">
        <string>Guess from data</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>To</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Offset</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QDoubleSpinBox" name="offsetDoubleSpinBox">
       <property name="minimum">
        <double>-99999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>99999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QDoubleSpinBox" name="toDoubleSpinBox">
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>99999.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>179</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
